src/shared/components/query/QueryStore.ts

```diff
@@ -1019,6 +1019,18 @@ export class QueryStore
 		500
 	);
 
+	readonly studyTags = remoteData<{[studyId:string]:any}>({
+		await: ()=>[this.cancerStudies],
+		invoke: async ()=>{
+			let studyTags: {[studyId:string]:any} = {};
+			for (const cancerStudy of this.cancerStudies.result) {
+				studyTags[cancerStudy.studyId] = await client.getTagsUsingGET({studyId: cancerStudy.studyId});

```

Comment:
i think this will result  in hunreds of separate network requests, one for each study.  can we get them in bulk by post?     are they important for all instances of portal or only for hyve?

Reply:
@alisman I'll work on creating a new endpoint which retrieves the data from all studies at once. Meanwhile you can check the updated description of this PR to understand a bit better the new functionality.

Reply:
@oplantalech can you please update, now that there is a new endpoint?

Reply:
@alisman @pieterlukasse Finally we're not using this function anymore, the data is only retrieved once the user hovers over the icon (check the current code).